.github/workflows/gh-pages.yml

```
@@ -8,6 +8,19 @@ on:
     - cron: "0 */4 * * *"
   workflow_dispatch:
 
+# Sets permissions of the GITHUB_TOKEN to allow deployment to GitHub Pages
+permissions:
+  contents: write
```

`zcash/zips` has this permission because it creates and pushes commits. This workflow doesn't, so it should only need:
```suggestion
  contents: read
```
(as I suggested in my review of the previous PR: https://github.com/zcash/developers/pull/82#discussion_r2260673260)